guestbook.module

```diff
@@ -0,0 +1,177 @@
+<?php
+module_load_include('inc', 'guestbook', 'include/guestbook_db');
+module_load_include('inc', 'guestbook', 'include/guestbook_core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+module_load_include('inc', 'guestbook', 'include/guestbook_weather');
+
+/**
+* @file
+* Build page, form for guestbook and weather block.
+*/
+
+/**
+* Implements hook_menu(). 
+*/
+ function guestbook_menu() {
+   $items = array();
```

Comment:
no necessary initialize array directly. It will be created in first definition. Remove this row